app/models/partner_company.rb

```diff
@@ -23,4 +23,17 @@ def add_employee(cpf_list)
     end
     invalid_cpfs
   end
+
+  def remove_employee(cpf_list)
+    invalid_cpfs = []
+    cpf_list.uniq!
+    cpf_list.each do |cpf|
+      if (cpf_to_remove = PartnerCompanyEmployee.find_by(cpf: cpf.strip))
+        PartnerCompanyEmployee.destroy(cpf_to_remove.id)
```

Comment:
Acho que essa lógica fica um pouco confusa por causa da atribuição da variável no `if`

O que acham de algo assim?

```suggestion
      cpf_to_remove = PartnerCompanyEmployee.find_by(cpf: cpf.strip)
      if cpf_to_remove.present?
        cpf_to_remove.destroy
```

Reply:
Daria até pra usar guard clause

Reply:
Usando guard clause assim 
`cpf_to_remove = PartnerCompanyEmployee.find_by(cpf: cpf.strip)
      cpf_to_remove.destroy if cpf_to_remove.present?

      invalid_cpfs << cpf.strip`
ele não passa pq da erro no `cpf_to_remove.destroy`, como poderiamos resolver?